docs/tutorials/articles/sales_dashboard/step_02/step_02.md

```diff
@@ -0,0 +1,198 @@
+```python
+        with tgb.part(class_name="card"):
+            with tgb.layout(columns="1 2 1"):
+                with tgb.part():
+                    tgb.text("Filter **From**", mode="md")
+                with tgb.part():
+                    tgb.text("Filter Product **Category**", mode="md")
+                with tgb.part(class_name="text-center"):
+                    tgb.button(
+                        "Apply",
+                    )
+```
+
+[Card](../../../../userman/gui/styling/stylekit.md) is a predefined style that will regroup visual elements in
+a white box. [layout](../../../../refmans/gui/viselements/generic/layout.md) allows you to create columns
+for visual elements. Here we create 3 columns with a ratio of 1:2:1, the second column will be twice as wide as the
+first and last columns. The contents of each column then needs to be separated in parts.
+
+![Layout](images/layout.png){ width=90% : .tp-image-border }
+
+We can now add [date selectors](../../../../refmans/gui/viselements/generic/date.md), 
+[selectors](../../../../refmans/gui/viselements/generic/selector.md) and a 
+[button](../../../../refmans/gui/viselements/generic/button.md) to apply the filters:
+
+```python
+        with tgb.part(class_name="card"):
+            with tgb.layout(columns="1 2 1"):
+                with tgb.part():
+                    tgb.text("Filter **From**", mode="md")
+                    tgb.date("{start_date}")
+                    tgb.text("To")
+                    tgb.date("{end_date}")
+                with tgb.part():
+                    tgb.text("Filter Product **Category**", mode="md")
+                    tgb.selector(
+                        value="{selected_category}",
+                        lov="{categories}",
+                        on_change=change_category,
+                        dropdown=True,
+                    )
+                    tgb.text("Filter Product **Subcategory**", mode="md")
+                    tgb.selector(
+                        value="{selected_subcategory}",
+                        lov="{subcategories}",
+                        dropdown=True,
+                    )
+                with tgb.part(class_name="text-center"):
+                    tgb.button(
+                        "Apply",
+                        class_name="plain apply_button",
+                        on_action=apply_changes,
+                    )
+```
+
+You'll notice we converted our selectors to dropdowns by setting the `dropdown` property to `True`.
+We also applied styling to the button: `plain` is a predefined style that colors the button in orange. 
+Predefined styles are available in visual elements documentation. 
+Check out the styling part of [button](../../../../refmans/gui/viselements/generic/button.md/).
+`apply_button` is a custom style that we can add using our CSS file:
+
+```css
+.apply_button {
+    margin-top: 158px;
+}
+```
+
+This will add a margin to the top of the button to align it with the filters.
```

Comment:
I think this will be now resolved in develop

Reply:
This is not the case for this one. The margin is a big one to align the widgets
![image](https://github.com/user-attachments/assets/c7cab411-5bff-4797-bce3-5379b7702568)

develop will probably force me to change the margin pixel value though. Is there a better way to align the button to the other widgets?